### DISCUSSION TITLE
What's most important to YOU?

### DISCUSSION THREAD

**Discussion by VeryKross** [VeryKross/BGBuddy#3] [Ideas]
I have a number of feature updates in the work, mostly dealing with having a visual (red LED) and audible (buzzer) alert for various thresholds that BGBuddy can track (high/low BG level, extended missed data, low phone battery, etc.). These will all be configurable on the BGBuddy.

Let me know what sorts of alerts are most important to you or if there are other features you'd like to see added.

**Comment by lexhome** [comment]
missed data, low phone battery
